imap_processing/tests/mag/test_mag_decom.py

```diff
@@ -1,72 +1,54 @@
-import dataclasses
 from pathlib import Path
 
-from imap_processing.mag.l0.decom_mag import decom_packets
+import pandas as pd
 
+from imap_processing.cdf import global_attrs
+from imap_processing.mag.l0.decom_mag import decom_packets, export_to_xarray
 
-def test_burst_decom():
-    current_directory = Path(__file__).parent
-    burst_test_file = current_directory / "mag_single_burst.pkts"
-    l0 = decom_packets(str(burst_test_file))[0]
 
-    assert l0.ccsds_header.PKT_APID == 1068
-    assert l0.ccsds_header.SRC_SEQ_CTR == 1
-    assert l0.ccsds_header.PKT_LEN == 1821
+def test_mag_decom():
+    current_directory = Path(__file__).parent
+    burst_test_file = current_directory / "mag_l0_test_data.pkts"
+    l0 = decom_packets(str(burst_test_file))
```

Comment:
I think you could leave this as a path being passed in so you don't have to cast to string in all of these.